src/main.rs

```diff
@@ -404,6 +417,22 @@ INCLUDE device.x
         );
     }
 
+    #[test]
+    fn test_perform_addition_hex_and_number() {
+        const ADDITION: &str = "0x20000000 + 1000";
+        let expected: u64 = 0x20000000 + 1000;
+
+        assert_eq!(perform_addition(ADDITION), expected);
+    }
+
+    #[test]
+    fn test_perform_addition_returns_number() {
+        const NO_ADDITION: &str = "0x20000000";
+        let expected: u64 = 536870912; //0x20000000 base 10
```

Comment:
You *could* just use the hex number.
```suggestion
        let expected: u64 = 0x20000000;
```

Reply:
... I know but I wanted to prove to myself it was true 😆 . Do you prefer me to use the hex number?